### PR TITLE
Provide access to the SiteApplication.

### PR DESCRIPTION
**Pull request by keithburgoyne**
... by making it a protected property on StoreAnalyticsTracker.

### CODE REVIEW

**Comment by keithburgoyne** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;rzNdaa5C&#x2F;1420-0-5d-upgrade-to-latest-facebook-pixel-code"> 0.5d - Upgrade to latest Facebook Pixel code.</a>

File: Store/StoreAnalyticsOrderTracker.php [hunk 1]
```diff
@@ -31,8 +36,9 @@ class StoreAnalyticsOrderTracker
 	// }}}
 	// {{{ public function __construct()
 
-	public function __construct(StoreOrder $order, $affiliation = null)
+	public function __construct(SiteApplication $app, StoreOrder $order, $affiliation = null)
```
Comment (isaacgrant) [inline Store/StoreAnalyticsOrderTracker.php]: Why is this needed?

Comment (keithburgoyne) [inline Store/StoreAnalyticsOrderTracker.php]: Yep. It's needed because CourseHost needs access to the application when disseminating the order. See https://github.com/silverorange/course-host/pull/535/files#diff-72d3b734d3dc38d1d3c19eecc130712aR62 for where it's used.

If there's a better way of doing this, I'm all ears.

**Comment by keithburgoyne** [comment]
We've decided this approach is unnecessary and have implemented an easier work around in code.
